blackbox/docs/administration/ssl.txt

```diff
@@ -102,15 +102,13 @@ Last but not least the connection parameter ``ssl=true``
 must be added to the connection URL so that the JDBC driver
 will try and establish an SSL connection.
 
+For further information visit `jdbc ssl documentation`_.
+
 Connect to a CrateDB node using psql
 ....................................
 
-By default ``psql`` attempts to use ssl if available on the node. In order to
-enforce certificate verification add ``sslmode=verify-full`` in your connection
-string. E.g.::
-
-    psql "sslmode=verify-full host=localhost port=5432"
-
-For further information visit `psql documentation`_.
+By default ``psql`` attempts to use ssl if available on the node. For further
+information visit `psql documentation`_.
```

Comment:
I would drop a note like "For further information also regarding the different ssl modes please visit..."